CRT.c

```diff
@@ -1185,6 +1187,8 @@ static void print_backtrace(void) {
 
    unsigned int item = 0;
 
+   char err_buf[512];
```

Comment:
Given symbol and file names may get quite large, consider more space here …

Reply:
Doubled the buffer